Update create-documentdb-dotnet.md

wayneadams opened the pull request:
Updates lines numbers of snippets to match code base
Updates snippet to match code base

azurecla commented:
Hi __@crystk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

crystk commented:
Modifying wording based on feedback from Dave Lubash

wayneadams committed:
Update create-documentdb-dotnet.md

Updates lines numbers of snippets to match code base
Updates snippet to match code base
